doc/source/data/preprocessors.rst

```diff
@@ -15,15 +22,7 @@ Ray AIR provides several common preprocessors out of the box and interfaces to d
 Overview
 --------
 
-The most common way of using a preprocessor is by passing it as an argument to the constructor of a Ray Train :ref:`Trainer <train-docs>` in conjunction with a :ref:`Ray Data dataset <data>`.
-For example, the following code trains a model with a preprocessor that normalizes the data.
-
-.. literalinclude:: doc_code/preprocessors.py
-    :language: python
-    :start-after: __trainer_start__
-    :end-before: __trainer_end__
-
-The  ``Preprocessor`` class with four public methods that can we used separately from a trainer:
+The  ``Preprocessor`` class has four public methods:
 
 #. ``fit()``: Compute state information about a :class:`Dataset <ray.data.Dataset>` (e.g., the mean or standard deviation of a column)
```

Comment:
I know this was carried over from the existing docs, but it would be great if we can update these to use the :class: and :meth: directives when appropriate.

Reply:
I'm going to cut scope here and say wontfix here, sorry.

Reply:
can we do it in a follow up?

Reply:
I don't have time to do this; I would suggest making an issue and finding an owner

Reply:
I updated your PR. Took ~1 minute.